src/crypto/CHIPCryptoPALOpenSSL.cpp

```diff
@@ -812,26 +822,42 @@ void ClearSecretData(uint8_t * buf, uint32_t len)
         }                                                                                                                          \
     } while (0)
 
+typedef struct Spake2p_Context
+{
+    EC_GROUP * curve;
+    BN_CTX * bn_ctx;
+    const EVP_MD * md_info;
+} Spake2p_Context;
+
+Spake2p_P256_SHA256_HKDF_HMAC::Spake2p_P256_SHA256_HKDF_HMAC(void) :
+    Spake2p(kP256_FE_Length, kP256_Point_Length, kSHA256_Hash_Length)
+{
+    memset(mSpake2pContext, 0, sizeof(mSpake2pContext));
```

Comment:
I think this is redundant, C++ constructors zero-initialize array member variables